### PR TITLE
Correct instructions, grammar for debug menu

### PR DESCRIPTION
**Pull request by mark7**
The CONTRIBUTING.md file gave incorrect instructions to bring up the
debug menu.  It also had some grammatical errors.  Fix.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -252,7 +252,7 @@ And ??? is the number. This automatically closes the issue when the PR is pulled
 
 Whether you are implementing a new feature or whether you are fixing a bug, it is always a good practice to test your changes in-game. It can be a hard task to create the exact conditions by playing a normal game to be able to test your changes which is why there is a debug menu. There is no default key to bring up the menu so you will need to assign one first.
 
-Bring up the keybindings menu (Press `?` than `1`), scroll down almost to the bottom and press `+` to add a new key binding. Press the letter that correspond to the *Debug menu* item than press the key you want to use to bring up the debug menu. To test your changes, create a new world with a new character. Once you are in that world, press the key you just assigned for the debug menu and you should see something like this:
```
Comment (Night-Pryanik) [inline CONTRIBUTING.md]: You can also de-capitalize "Press" in the first line.

Comment (mark7) [inline CONTRIBUTING.md]: Good point

**Comment by mark7** [comment]
Decided to cover the whole file while I was at it.

**Comment by mark7** [comment]
Hmm.  Looks like this is not related to my patch.  Is there a way to retrigger the checks?

**Pull request merged**